build.sbt

```diff
@@ -429,12 +677,13 @@ lazy val scalaXml = libraryProject("scala-xml")
     description := "Provides scala-xml codecs for http4s",
     startYear := Some(2014),
     libraryDependencies ++= Seq(
-      Http4sPlugin.scalaXml,
-    ),
+      Http4sPlugin.scalaXml.value
+    )
   )
   .dependsOn(core, testing % "test->test")
 
-lazy val twirl = http4sProject("twirl")
+// Full cross helps workaround issues with twirl directories
+lazy val twirl = http4sProject("twirl", CrossType.Full, List(JVMPlatform))
```

Comment:
Is this going to publish as `http4s-twirl_2.13.6` instead of `http4s_twirl_2.13`?  I'd very much like to avoid that, if I understand it right.  What changed?

Reply:
Umm, yes? E.g. see https://search.maven.org/artifact/com.armanbilge/http4s-twirl_2.13/1.0-377-020cf9e/.

But also, that _is_ how _you_ publish it no?

https://search.maven.org/artifact/org.http4s/http4s-twirl_2.12

The `sbt-crossproject` plugin is really bad with "non-standard" source directories, e.g. see my PRs to add support for literally the basics. Doing a full cross seems to disappear these problems.

https://github.com/portable-scala/sbt-crossproject/pull/132
https://github.com/portable-scala/sbt-crossproject/pull/133

Reply:
BTW, we can totally revert this to the old way of doing things if we separate out `http4sProject(...)` and `http4sCrossProject(...)` for example. The nice thing about everything being a cross project is that you can programmatically identify which platforms each module supports, this is how I make the `rootJVM` and `rootJS for example.

Reply:
No, it doesn't look like it's doing what I was concerned about.  I thought it was putting the full Scala version in the artifact name instead of the binary version: _2.13.6 vs. 2.13.  That's not how we do things, but it's also not how your snapshot does things.  I misunderstand what that setting does, but it's not doing the wicked thing I thought it was, so I think it's fine.